lib/generators/alchemy_i18n/install/install_generator.rb

```diff
@@ -53,12 +53,6 @@ def append_manifest
         end
       end
 
-      def add_rails_i18n
-        environment do
-          "config.i18n.available_locales = #{locales.map(&:to_sym).inspect}"
-        end
-      end
-
```

Comment:
With that removed all locales will be loaded into RAM, making most of the changes we made recently obsolete.

Reply:
I see the issue with the `en` locale, but maybe we address that instead?

Reply:
Done in #45